implementations/rust/ockam/ockam_command/src/operation/util.rs

```diff
@@ -0,0 +1,56 @@
+use std::io::Write;
+
+use anyhow::anyhow;
+use colorful::Colorful;
+use tokio_retry::strategy::FixedInterval;
+use tokio_retry::Retry;
+
+use ockam_api::cloud::operation::Operation;
+
+use crate::util::api::CloudOpts;
+use crate::util::{api, RpcBuilder};
+use crate::{CommandGlobalOpts, Result};
+
+pub async fn check_for_completion<'a>(
+    ctx: &ockam::Context,
+    opts: &CommandGlobalOpts,
+    cloud_opts: &CloudOpts,
+    api_node: &str,
+    operation_id: &str,
+) -> Result<()> {
+    let total_sleep_time_ms = 10 * 60 * 1000;
```

Comment:
Should this be a const and/or defaultable CLI flag?

Reply:
I'm not sure if we have a standard yet to make this a CLI Flag; Making it a const sounds good though.